application/core/workflow.py

```diff
@@ -135,3 +139,16 @@ def csv_to_json(csv_file):
                 json_data.append(row)
 
     return json_data
+
+
+def updateColumnFieldLog(column_field_log, required_fields):
+    for field in required_fields:
+        if not any(entry["field"] == field for entry in column_field_log):
+            column_field_log.append({"field": field, "missing": True})
```

Comment:
if the field is in the column field log, should we add this missing param here?
not sure if this is impossible or not?